### PR TITLE
Add description of CRD resources displayed in RRV plugin

### PR DESCRIPTION
**Pull request by obabativavmware**
Which other branches should this be merged with (if any)?
- 1.2

### CODE REVIEW

**Comment by chinigo** [comment]
Some copyedits:

```
### <a id="resources-included"></a> Resources

These built-in Kubernetes resources are included in this view:

- Services
- Deployments
- ReplicaSets
- Pods

The Runtime Resource Visibility plug-in will also display CRDs created with the [Supply Chain](./scc-tap-gui.md), including:

- Cartographer Workloads
- Knative Services, Configurations, Revisions, and Routes

This example ilustrates how CRDs from Supply Chain (yellow) are associated with Knative Resources (orange) and built-in ones (red).

![Screenshot of OwnerShip card](images/runtime-resources-crd-hierarchy.png)
```

**Comment by RichardJJG** [comment]
@obabativavmware, what does "ones" in [this sentence](https://github.com/pivotal/docs-tap/pull/1206/files#diff-35e2be7cc62b3b299314d619801df04a683fc7896972b85a228fce60cd4f068bR56) refer to? CRDs or resources?

**Comment by chinigo** [comment]
Good catch. "One" refers to "resources" here. Maybe rephrase to:

> This example illustrates how CRDs from Supply Chain (yellow) are associated with Knative Resources (orange) and core Kubernetes resources (red).

(Also looks like there's a typo in the word "ilustrates" [sic].)

**Comment by RichardJJG** [comment]
> Good catch. "One" refers to "resources" here. Maybe rephrase to:
> 
> > This example illustrates how CRDs from Supply Chain (yellow) are associated with Knative Resources (orange) and core Kubernetes resources (red).
> 
> (Also looks like there's a typo in the word "ilustrates" [sic].)

Thanks, and, yeah, I caught the typo.

**Comment by chinigo** [comment]
Oh blerg just noticed the inconsistency in casing in my recommendation, too. I think _probably_ we'd want to use a lower-case for "resources," yeah? So:

> This example illustrates how CRDs from Supply Chain (yellow) are associated with Knative resources (orange) and core Kubernetes resources (red).

**Comment by RichardJJG** [comment]
I've edited the image, too, @chinigo. You can see all the edits here:
https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.2/tap/GUID-tap-gui-plugins-runtime-resource-visibility.html#resources-included

(Images need to be self-explanatory and key information should not exist solely in an image so as not to discriminate against blind people)

**Comment by chinigo** [comment]
Image is much clearer now. And that's a good policy, I'll keep it mind for future PRs from my team.

Thanks!
